### DISCUSSION TITLE
[1992860]Vetrix Worlds [仅本体]

### DISCUSSION THREAD

**Discussion by wanghaoyuas** [pjy612/SteamManifestCache#128] [Share]
` appid:1992860` 
keys: 
```
1992861;2a7b35a888175a80463211e91e7645068379304d6cf093624f1860b33831ef9e
```
清单[1992861_978208300455413886.zip](https://github.com/pjy612/SteamManifestCache/files/13758178/1992861_978208300455413886.zip)
内测码:0E14F670C5691215
邮箱:yuanyuanwhy@vip.qq.com

**Comment by pjy612** [comment]
查收
